deployment/graph/otp

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec java -Xmx"$JAVA_MX" -jar /usr/local/share/java/otp.jar "$@"
```

Comment:
I noticed when building the graph that I received a warning that `build-config.json` and `router-config.json` were absent and that default configuration would be used. I searched the repo to see if we were using either of those and found a [template for `router-config.json`](https://github.com/azavea/cac-tripplanner/blob/develop/deployment/ansible/roles/cac-tripplanner.otp-data/templates/router-config.json.j2) which sets up the configuration for using bike-sharing stations. However, that seems like it's only a run-time configuration and [isn't used when building the graph](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/docs/Configuration.md#three-scopes-of-configuration).

So I don't think we need to work to include the `router-config.json` during the graph build step here, but we should thoroughly test the bike-sharing functionality the first time we deploy using this graph building process, and we'll need to include that file if we ever work on #1275.